portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/SharedComponents/AttachedPolicyCard.tsx

```diff
@@ -0,0 +1,155 @@
+import { Alert } from 'AppComponents/Shared';
+import API from 'AppData/api.js';
+import Utils from 'AppData/Utils';
+import { FormattedMessage } from 'react-intl';
+import ApiContext from 'AppComponents/Apis/Details/components/ApiContext';
+import AttachedPolicyCardShared from 'AppComponents/Shared/PoliciesUI/AttachedPolicyCard';
+import type { AttachedPolicy, PolicySpec } from '../Types';
+import PolicyConfigurationEditDrawer from '../GlobalSpecificComponents/PolicyConfigurationEditDrawer';
+import GlobalPolicyContext from '../GlobalPolicyContext';
+
+interface AttachedPolicyCardProps {
+    policyObj: AttachedPolicy;
+    currentPolicyList: AttachedPolicy[];
+    setCurrentPolicyList: React.Dispatch<React.SetStateAction<AttachedPolicy[]>>;
+    currentFlow: string;
+    verb: string;
+    target: string;
+    allPolicies: PolicySpec[] | null;
+    isAPILevelPolicy: boolean;
+}
+
+/**
+ * Renders a single sortable policy card.
+ * @param {any} AttachedPolicyCardProps Input props from parent components.
+ * @returns {TSX} Sortable attached policy card UI.
+ */
+const AttachedPolicyCard: FC<AttachedPolicyCardProps> = ({
+    policyObj,
+    currentPolicyList,
+    setCurrentPolicyList,
+    currentFlow,
+    verb,
+    target,
+    allPolicies,
+    isAPILevelPolicy,
+}) => {
+    
+    const { api } = useContext<any>(ApiContext);
+    const { deleteGlobalOperation } = useContext<any>(GlobalPolicyContext);
+    const [drawerOpen, setDrawerOpen] = useState(false);
+    
+
+    /**
+     * Handle policy delete
+     * @param {React.MouseEvent<HTMLButtonElement, MouseEvent>} event event
+     */
+    const handleDelete = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        const filteredList = currentPolicyList.filter(
+            (policy) => policy.uniqueKey !== policyObj.uniqueKey,
+        );
+        const policyToDelete = currentPolicyList.find(
+            (policy) => policy.uniqueKey === policyObj.uniqueKey,
+        );
+        setCurrentPolicyList(filteredList);
+        deleteGlobalOperation(
+            policyToDelete?.uniqueKey,
+            currentFlow,
+        );
+        event.stopPropagation();
+        event.preventDefault();
+    };
+
+    /**
+     * Handle policy download
+     * @param {React.MouseEvent<HTMLButtonElement, MouseEvent>} event event
+     */
+    const handlePolicyDownload = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        event.stopPropagation();
+        event.preventDefault();
+        if (policyObj.isAPISpecific) {
+            const globalPolicyContentPromise = API.getOperationPolicyContent(
+                policyObj.id,
+                api.id,
+            );
+            globalPolicyContentPromise
+                .then((globalPolicyResponse) => {
+                    Utils.forceDownload(globalPolicyResponse);
+                })
+                .catch((/* error */) => {
+                    // console.error(error);
```

Comment:
```suggestion
                    
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)